src/ec2/prompter.ts

```diff
@@ -3,28 +3,36 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { createQuickPick, QuickPickPrompter } from "../shared/ui/pickerPrompter"
-import { DataQuickPickItem } from "../shared/ui/pickerPrompter"
-import { AsyncCollection } from "../shared/utilities/asyncCollection"
-import { isValidResponse } from '../shared/wizards/wizard'
+import { RegionSubmenu, RegionSubmenuResponse } from '../shared/ui/common/regionSubmenu'
+import { getInstanceIdsFromRegion } from "./utils";
+import { DataQuickPickItem } from "../shared/ui/pickerPrompter";
 
-function asQuickpickItem(stringItem: string): DataQuickPickItem<string>[] 
-{
-    return [{
-        label: stringItem, 
-        data: stringItem
-    }]
+type EC2InstanceId = string
```

Comment:
I would avoid aliasing primitives because it gives a false sense of robustness. Let identifiers (properties, parameters, etc.) describe the expected shape instead.